python/lsst/pipe/drivers/skyCorrection.py

```diff
@@ -54,21 +130,40 @@ def setDefaults(self):
         self.bgModel2.ySize = 256
         self.bgModel2.smoothScale = 1.0
 
+        self.quantum.dimensions = ("instrument", "visit")
 
-class SkyCorrectionTask(BatchPoolTask):
+
+class SkyCorrectionTask(pipeBase.PipelineTask, BatchPoolTask):
     """Correct sky over entire focal plane"""
     ConfigClass = SkyCorrectionConfig
     _DefaultName = "skyCorr"
 
+    def adaptArgsAndRun(self, inputData, inputDataIds, outputDataIds, butler):
+        inputData.pop("rawLinker", None)
+        return super().adaptArgsAndRun(inputData, inputDataIds, outputDataIds, butler)
+
+    @classmethod
+    def getPrerequisiteDatasetTypes(cls, config):
+        names = set(cls.getInputDatasetTypes(config))
+        names.remove("calExpArray")
+        names.remove("calBkgArray")
+        names.remove("rawLinker")
+        return names
```

Comment:
I think is is better to declare what you do want and not what you dont in this case